packages/vitest/src/runtime/workers/utils.ts

```diff
@@ -27,16 +25,3 @@ export function createForksRpcOptions(nodeV8: typeof import('v8')): WorkerRpcOpt
     },
   }
 }
-
-function parsePossibleRegexp(str: string | RegExp) {
```

Comment:
I would be cautious to mix thread level utils and workers utils, rollup can bundle them together one time and we won't notice

Reply:
Yep I agree. I'll refactor the d7c319e4af94cac7a1361523c989aa6f0b13476f once more.